src/etl/ETLHelpers.h

```diff
@@ -179,4 +179,22 @@ getMarkers(size_t numMarkers)
     return markers;
 }
 
-#endif
+template <class F>
+void
+synchronous(F&& f)
+{
+    boost::asio::io_context ctx;
+    std::optional<boost::asio::io_context::work> work;
+
+    work.emplace(ctx);
+
+    boost::asio::spawn(ctx, [&f, &work](boost::asio::yield_context yield) {
+        f(yield);
+
+        work.reset();
+    });
+
+    ctx.run();
+}
+
+#endif  // RIPPLE_APP_REPORTING_ETLHELPERS_H_INCLUDED
```

Comment:
I'm in favor of replacing include guards with `#pragma once`. This is not necessary for this pull request, but we could note it as technical debt if everyone is on board with switching over. However, this does conflict with the convention in rippled. @cjcobb23 - thoughts?

Reply:
Are there any downsides to `#pragma once`?

Reply:
I mean there are some issues, we can forget this for now. Resolving.